### PR TITLE
fix: Changed key words to camelCase

### PR DESCRIPTION
**Pull request by khalsz**

### CODE REVIEW

File: ukaea-schema/diagnostics/camera.schema.json [hunk 1]
```diff
@@ -3,14 +3,14 @@
     "$id": "urn:ukaea:experiment:hive:diagnostics:camera-schema",
     "description": "Metadata to describe camera diagnostics",
     "type": "object",
-    "required": ["camera_information", "lens_information", "capture_settings"],
+    "required": ["cameraInformation", "lensN", "captureSettings"],
```
Comment (NathanCummings) [inline ukaea-schema/diagnostics/camera.schema.json:6]: Should this be "lensInformation"?

Comment (khalsz) [reply ukaea-schema/diagnostics/camera.schema.json:6]: Yes, 
Nathan. I have changed it now. I will add the MR

File: ukaea-schema/facility/hive/port.schema.json [hunk 2]
```diff
@@ -19,7 +19,7 @@
         },
         "portSizeStandard": {
             "type": "string",
-            "example": "CF100"
+            "example": "CF100"
```
Comment (adam-parker1) [inline ukaea-schema/facility/hive/port.schema.json:22]: I am guessing this has been run through a linter/JSON validator, we probably want to remove this whitespace

Comment (NathanCummings) [reply ukaea-schema/facility/hive/port.schema.json:22]: This suggests to me that #10 should be next on the agenda.

**Comment by adam-parker1** [comment]
Fixes issue #9

File: ukaea-schema/diagnostics/camera.schema.json
Comment (adam-parker1) [inline ukaea-schema/diagnostics/camera.schema.json:1]: This commit has reversed the changes we took from develop and has reverted them back

**Pull request merged**